examples/32_calibration_reader.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
```

Comment:
this example should do what it says: `calibration_reader`
So remove NN, rgb and others, since they overlap with the other examples.
It should only read the calibration info, and print it.

Another option would be also doing something with the information, like doing rectification in host and comparing with device side implementation.
Same for cpp